src/pathSatisfies.js

```diff
@@ -0,0 +1,31 @@
+var _curry3 = require('./internal/_curry3');
+var path = require('./path');
+
+
+/**
+ * Returns `true` if the specified object property at given path satisfies the
+ * given predicate; `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @sig (a -> Boolean) -> [String] -> Object -> Boolean
+ * @param {Function} pred
+ * @param {Array} propPath
+ * @param {*} obj
+ * @return {Boolean}
+ * @see R.propSatisfies, R.path
+ * @example
+ *
+ *      R.pathSatisfies(y => y > 0, ['x', 'y'], {x: {y: 2}}); //=> true
+ */
+module.exports = _curry3(function pathSatisfies(pred, propPath, obj) {
+  switch (propPath.length) {
+    case 0:
+      return false;
+    case 1:
+      return pred(obj[propPath[0]]);
```

Comment:
This case is unnecessary. The case below can cover `propPath.length === 1` as well.